KwcNotificationBox/Kwc/Box/Component.defer.js

```diff
@@ -0,0 +1,45 @@
+"use strict";
+var $ = require('jquery');
+var onReady = require('kwf/on-ready');
+
+onReady.onRender('.kwcClass', function (el, config) {
+
+    var setCookieValue = function(key, value, daysUntilExpired) {
+        var d = new Date();
+        d.setTime(d.getTime() + (daysUntilExpired * 24 * 60 * 60 * 1000));
+        var expires = 'expires=' + d.toUTCString();
+        document.cookie = key + '=' + value + ';' + expires + ';path=/';
+    };
+
+    var getCookieValue = function(key) {
+        var name = key + '=',
+            decoded = decodeURIComponent(document.cookie),
+            ca = decoded.split(';');
+        for (var i = 0; i <ca.length; i++) {
+            var c = ca[i];
+            while (c.charAt(0) == ' ') {
+                c = c.substring(1);
+            }
+            if (c.indexOf(name) == 0) {
+                return c.substring(name.length, c.length);
+            }
+        }
+        return '';
+    };
```

Comment:
Statt setCookieValue und getCookieValue das npm-paket js-cookie verwenden (so wie in /koala-framework/commonjs/cookie-opt/adapter.js)